### DISCUSSION TITLE
Integration with the iOS files app

### DISCUSSION THREAD

**Discussion by r8plicant** [isontheline/pro.webssh.net#1034] [Ideas]
Hi, it would be nice as a new feature to get integration with the iOS Files app (similarly as Termium does). This would allow us to see configured SFTP servers and easily move files around from/to those servers and other Files App servers (i.e. iCloud, Google Drive, local iOS files, SMB, etc.).
Thanks for this great app!

**Comment by isontheline** [comment]
Hi @r8plicant 👋

Thank you so much for reaching me out and for your support 🙏

You're right this could be a great feature!
An issue has been opened 13 months ago about it : https://github.com/isontheline/pro.webssh.net/issues/787

I need to find time to implement it

I wish you a nice day ☀️

**Comment by r8plicant** [comment]
Ah I didn’t see it when searching on GitHub. Good, thank you!
